utils/build_ds_container.sh

```diff
@@ -72,7 +82,39 @@ if [ "$namespace" == "openshift-compliance" ]; then
     oc apply -f "$root_dir/ocp-resources/compliance-operator-ns.yaml"
 fi
 
-if [ "$build_in_cluster" == "false" ];then
+if [ -z "$imagerepo" ]; then
+    if [ "$build_in_cluster" == "false" ];then
+        # build the product's content
+        "$root_dir/build_product" ${products[@]} "${params[@]}"
+        result=$?
+
+        if [ "$result" != "0" ]; then
+            echo "Error building content"
+            exit $result
+        fi
+
+        # Create buildconfig and ImageStream
+        # This enables us to create a configuration so we can build a container
+        # with the datastream
+        # If they already exist, this is not a problem
+        oc apply -n "$namespace" -f "$root_dir/ocp-resources/ds-from-local-build.yaml"
+
+        # Create output directory
+        from_dir=$(mktemp -d)
+
+        # Copy datastream files to output directory
+        cp "$root_dir/build/"*-ds.xml "$from_dir"
```

Comment:
Do we still need those, since we are just copying from the build folder in Dockerfile?

Reply:
we do, this isn't for the registry push, but for the OpenShift push instead

Reply:
Ahh, I think it is still used here https://github.com/ComplianceAsCode/content/blob/7b64c1e178e8d820894ccf7ba55fcb8f962716f5/utils/build_ds_container.sh#L161